hkmc2/shared/src/main/scala/hkmc2/codegen/Lowering.scala

```diff
@@ -304,7 +302,22 @@ class Lowering(using TL, Raise, Elaborator.State):
         term(finallyDo)(_ => End()),
         k(Value.Ref(l))
       )
-    
```

Comment:
Let's keep these BbML-specific cases grouped up with empty lines surrounding them, and ideally a comment on top.